docs/getting-started.md

```diff
@@ -1,372 +1,273 @@
+or run Firecracker as `root` (via `sudo`).
 
-If you'd like to boot up a guest machine without using the API socket, you can
-do that by passing the parameter `--config-file` to the Firecracker process.
-The command for starting Firecracker with this option will look like this:
+You can check if you have access to `/dev/kvm` with:
 
-```wrap
-./firecracker --api-sock /tmp/firecracker.socket --config-file <path_to_the_configuration_file>
+```bash
+[ -r /dev/kvm ] && [ -w /dev/kvm ] && echo "OK" || echo "FAIL"
 ```
 
-`path_to_the_configuration_file` should represent the path to a file that
-contains a JSON which stores the entire configuration for all of the microVM's
-resources. The JSON **must** contain the configuration for the guest kernel and
-rootfs, as these are mandatory, but all of the other resources are optional,
-so it's your choice if you want to configure them or not. Because using this
-configuration method will also start the microVM, you need to specify all
-desired pre-boot configurable resources in that JSON. The names of the
-resources are the ones from the `firecracker.yaml` file and the names of
-their fields are the same that are used in API requests. You can find an
-example of configuration file at `tests/framework/vm_config.json`.
-After the machine is booted, you can still use the socket to send
-API requests for post-boot operations.
-
-## Building From Source
-
-The quickest way to build and test Firecracker is by using our development
-tool ([`tools/devtool`](../tools/devtool)). It employs a
-per-architecture [Docker container](../tools/devctr/)  to store the software toolchain
-used throughout the development process. If you need help setting up
-[Docker](https://docker.com) on your system, you can check out
-[Appendix B: Setting Up Docker](#appendix-b-setting-up-docker).
-
-### Getting the Firecracker Sources
+## Running Firecracker
 
-Get a copy of the Firecracker sources by cloning our GitHub repo:
+In production, Firecracker is designed to be run inside
+an execution jail, set up by the [`jailer`](../src/jailer/) binary. This is how
+our [integration test suite](#running-the-integration-test-suite) does it. This
+guide will not use the [`jailer`](../src/jailer/).
 
 ```bash
-git clone https://github.com/firecracker-microvm/firecracker
-```
+ARCH="$(uname -m)"
 
-All development happens on the main branch and we use git tags to mark
-releases. If you are interested in a specific release (e.g. v0.10.1), you can
-check it out with:
+# Download a linux kernel binary
+wget https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/${ARCH}/kernels/vmlinux.bin
 
-```bash
-git checkout tags/v0.10.1
-```
+# Download a rootfs
+wget https://s3.amazonaws.com/spec.ccfc.min/ci-artifacts/disks/${ARCH}/ubuntu-18.04.ext4
 
-### Building Firecracker
+# Download the ssh key for the rootfs
+wget https://s3.amazonaws.com/spec.ccfc.min/ci-artifacts/disks/${ARCH}/ubuntu-18.04.id_rsa
```

Comment:
ah, didnt know this was something that needed to be done. explains all my difficulties ssh-ing into my microvms 😂